en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+        | **Type**          | **other**                  |     
+        | **Other Type**    | `map<string[]>`            |
+        | **Name**          | `queryParams`              |
+        | **Expression**    | `request.getQueryParams()` |
+
+    3. Click **Save**.
+    
+6. To enable Choreo to read the value for the `item` parameter in the order request, add another variable statement as follows:
+
+    1. Click the **+** below the `queryParams` variable statement you added, and then click **Variable**. 
+    
+    2. Enter information for the variable as follows:
+
+        | **Field**         | **Value**                    |
+        |-------------------|------------------------------|
+        | **Type**          | **string**                   |     
+        | **Name**          | `inventoryItemId`            |
+        | **Expression**    | `queryParams.get("item")[0]` |
+    
+    3. Click **Save**.
+    
+7. To enable Choreo to read the value for the `quantity` parameter in the order request, add another variable statement as follows:
+
+    1. Click the **+** icon below the `inventoryItemId` variable statement you added, and then click **Variable**. 
+    
+    2. Enter information for the variable as follows:
+
+        | **Field**         | **Value**                                           |
+        |-------------------|-----------------------------------------------------|
+        | **Type**          | **int**                                             |     
+        | **Name**          | `quantity`                                          |
+        | **Expression**    | `check int:fromString(queryParams.get("count")[0])` |
+        
+    3. Click **Save**.
+     
+ 8. To connect to the Google sheet where inventory records are maintained, add an API call as follows: 
+            
+    1. Click the **+** below the `quantity` variable statement you added, and then click **API Calls**.
+    
+    2. Click **Google Sheets**
+    
+    3. Click **Connect to Google Sheets**, and then click **Proceed**. Click on the Google Account of your choice and click **Allow** too allow Choreo to access it as described in the page. Then click **Save**.
+    
+    4. Click the **+** icon below the Google Sheets connection you just added. Click **API Calls**, and then click **sheetsEndpoint** under **Choose existing connection**.
+    
+    5. In the **Operation** field, select **Get values from a column**. Then enter information as follows in the rest of the fields that appear:
+    
+        | **Field**                  | **Value**                                                                                                       |
+        |----------------------------|-----------------------------------------------------------------------------------------------------------------|
+        | **SpreadsheetId**x         | The ID of your google sheet.                                                                                    |  
```

Comment:
Here we need to have more details on what we mean by the ID of the Google sheet. The reader should be given specific details on how to find this ID value.

Reply:
Added an example to show how to derive it.